packages/sdk/src/constants/chain-properties.ts

```diff
@@ -0,0 +1,37 @@
+import { ChainId } from '../enums';
+
+export const CHAIN_LABEL: Record<ChainId, string> = {
+  [ChainId.ETHEREUM]: 'Ethereum',
+  [ChainId.ARBITRUM]: 'Arbitrum',
+  [ChainId.FANTOM]: 'Fantom',
+  [ChainId.MATIC]: 'Polygon',
+  [ChainId.OPTIMISM]: 'Optimism',
+  [ChainId.GNOSIS]: 'Gnosis',
+  [ChainId.GOERLI]: 'Goerli',
+  [ChainId.OPTIMISM_GOERLI]: 'Optimism Goerli',
+  [ChainId.MATIC_MUMBAI]: 'Mumbai',
+};
+
+export const CHAIN_NATIVE_TOKEN_NAME: Record<ChainId, string> = {
```

Comment:
Those are available as:
`NATIVE[chainId].name` or `NATIVE[chainId].symbol`

Reply:
Same as above, `NATIVE[chainId]` can be `undefined`, when `onboard` requires a value. Can I freely do `... ?? ""` since for our cases It will always have a value or should we change anything?

Reply:
no longer need if change on L42 in "packages/frontend/helpers/chains.ts" is accepted